components/composer/src/Composer.svelte

```diff
@@ -652,19 +655,25 @@
             <button
               type="button"
               bind:this={ccField}
-              hidden={!_this.show_cc &&
+              hidden={_this.show_cc &&
```

Comment:
Don't we want to hide this if `_this.show_cc` is false?

Reply:
When `_this.show_cc` is `true` (cc field is visible) this button should be hidden. Please take a look at the recording to see it in action.

Reply:
ohh sorry, this is the button 🤦 . My bad, I didn't expand the code to see what it was